core/src/main/java/gov/nasa/jpl/mudrod/main/MudrodEngine.java

```diff
@@ -333,24 +296,19 @@ public static void main(String[] args) {
     // boolean options
     Option helpOpt = new Option("h", "help", false, "show this help message");
 
-    // preprocessing + processing
+    // log ingest (preprocessing + processing)
+    Option logIngestOpt = new Option("l", LOG_INGEST, false, "begin log ingest");
+    // metadata ingest (preprocessing + processing)
+    Option metaIngestOpt = new Option("m", META_INGEST, false, "begin metadata ingest");
+    // ingest both log and metadata
     Option fullIngestOpt = new Option("f", FULL_INGEST, false, "begin full ingest Mudrod workflow");
-    // processing only, assuming that preprocessing results is in logDir
+    // processing only, assuming that preprocessing results is in dataDir
     Option processingOpt = new Option("p", PROCESSING, false, "begin processing with preprocessing results");
 
-    // import raw web log into Elasticsearch
-    Option logIngestOpt = new Option("l", LOG_INGEST, false, "begin log ingest without any processing only");
-    // preprocessing web log, assuming web log has already been imported
-    Option sessionReconOpt = new Option("s", SESSION_RECON, false, "begin session reconstruction");
-    // calculate vocab similarity from session reconstrution results
-    Option vocabSimFromOpt = new Option("v", VOCAB_SIM_FROM_LOG, false, "begin similarity calulation from web log Mudrod workflow");
-    // add metadata and ontology preprocessing and processing results into web
-    // log vocab similarity
-    Option addMetaOntoOpt = new Option("a", ADD_META_ONTO, false, "begin adding metadata and ontology results");
 
     // argument options
-    Option logDirOpt = OptionBuilder.hasArg(true).withArgName("/path/to/log/directory").hasArgs(1).withDescription("the log directory to be processed by Mudrod").withLongOpt("logDirectory")
-        .isRequired().create(LOG_DIR);
+    Option dataDirOpt = OptionBuilder.hasArg(true).withArgName("/path/to/data/directory").hasArgs(1).withDescription("the data directory to be processed by Mudrod").withLongOpt("logDirectory")
```

Comment:
The long description should change if the short description is changing.
```.withLongOpt("logDirectory")``` should be ```.withLongOpt("dataDirectory")```